app/lib/level-generation.js

```diff
@@ -706,7 +706,7 @@ generateProperty(null, function (level, parameters) {
     juniorPlayer.config = _.find(heroSource.components, (component) => component.original === defaultHeroComponentIDs.JuniorPlayer).config
   } else {
     juniorPlayer.config = {
-      programmableSnippets: ['for-loop', 'if'],
+      programmableSnippets: ['for-loop', 'if', '=='],
```

Comment:
@nwinter what do we consider `programmableSnippets`  here?

Reply:
This just controls the defaults for what methods/blocks show up for the player to use, when generating new levels. SInce I've moved onto generating `==` and `!=` levels, I always want those in the defaults list. A snippet is distinct from a method in that it's not in the form of `hero.foo`.